transform/README.md

```diff
@@ -0,0 +1,87 @@
+# Transforms: _**warp_coords**_
```

Comment:
Good but algorithm needs to be in its own folder to avoid overwriting others works.